setup1-contributor/src/objects.rs

```diff
@@ -39,6 +43,12 @@ pub struct LockResponse {
     /// The locator where the participant will upload their completed contribution.
     #[serde(alias = "responseLocator")]
```

Comment:
are the locators still necessary now that we are using the chunk id and contribution id?

Reply:
It's also good to check the integration test running over multiple rounds (like https://github.com/AleoHQ/aleo-setup-integration-test/blob/e78cee2ffc1fc6ea4512ab4e6d77435bfc3dbc26/dev-tests.ron#L314)  and seeing where the final verification for a chunk gets stored (it normally gets placed in the next round's directory, strange, I know), not sure if this will have an impact on that.

Reply:
We use it for the local storage paths. So it's still being used, but no longer for the server-side storage, only the client-side.

As for the integration tests, I have ran a few, but running the one with multiple rounds is a good idea. I've made sure to set all the upload paths to the right rounds (I noticed the verifier sent their solution into a folder that was a round up) so I'll run that test to make sure everything carries over correctly.

Reply:
The integration test you pointed out also ran perfectly fine, and reached the end of the ceremony without a hitch!